.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
       - name: Install Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: "16.x"
```

Maybe v18 could be used as it has a longer support window (until 2025).
```suggestion
          node-version: "18.x"
```